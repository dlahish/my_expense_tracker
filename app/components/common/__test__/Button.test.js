import React from 'react'
import 'react-native'
import Button from '../Button'

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(
    <Button children='Button Text'/>
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
