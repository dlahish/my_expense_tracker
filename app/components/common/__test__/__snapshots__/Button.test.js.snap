exports[`test renders correctly 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  hitSlop={undefined}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Array [
      Object {
        "backgroundColor": "transparent",
      },
      Array [
        Object {
          "alignSelf": "stretch",
          "borderRadius": 5,
          "flex": 1,
          "height": 44,
          "justifyContent": "center",
          "overflow": "hidden",
        },
        undefined,
      ],
    ]
  }
  testID={undefined}>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "fontSize": 18,
          "margin": 5,
          "textAlign": "center",
        },
        Object {
          "color": "#000",
        },
      ]
    }>
    Button Text
  </Text>
</View>
`;
